agent_shared/src/config.rs

```diff
@@ -245,6 +286,54 @@ pub enum SupportedDidMethod {
     IotaRms,
 }
 
+/// (A subset of) DID method traits. The methods follow a naming convention that expresses boolean predicates as verb
+/// phrases, as specified in the DID traits documentation:
+/// https://github.com/decentralized-identity/did-traits/blob/v0.8.0/schemas/v0.8.0/traits.json
+impl SupportedDidMethod {
+    pub fn supports_update(&self) -> bool {
+        match self {
+            SupportedDidMethod::Web
+            | SupportedDidMethod::Iota
+            | SupportedDidMethod::IotaSmr
+            | SupportedDidMethod::IotaRms => true,
+            SupportedDidMethod::Jwk | SupportedDidMethod::Key => false,
+        }
+    }
+
+    pub fn hosted_centrally(&self) -> bool {
+        match self {
+            SupportedDidMethod::Jwk
+            | SupportedDidMethod::Key
+            | SupportedDidMethod::Iota
+            | SupportedDidMethod::IotaSmr
+            | SupportedDidMethod::IotaRms => false,
+            SupportedDidMethod::Web => true,
+        }
+    }
+
+    pub fn decentrally_hosted(&self) -> bool {
```

Comment:
Should be `hosted_decentrally` as defined in **DID Traits v0.8.0**

Reply:
fixed